build.gradle

```diff
@@ -2,22 +2,21 @@
 
 buildscript {
 
-    ext.kotlin_version = '1.2.41'
-    ext.android_plugin_version = '3.1.2'
-    ext.targetSdkVersion = 27
-    ext.compileSdkVersion = 27
-    ext.buildToolsVersion = '27.0.3'
+    ext.kotlin_version = '1.2.71'
+    ext.android_plugin_version = '3.2.0'
+    ext.targetSdkVersion = 28
+    ext.compileSdkVersion = 28
     ext.minSdkVersion = 14
     ext.novodaBintrayVersion = '0.8.1'
-    ext.supportLibVersion = '27.1.1'
-    ext.databinding_version = '3.1.1'
+    ext.androidX = "1.0.0"
```

Comment:
I'd suggest keeping consistency for naming version variables here :)
That would mean to:
- Keep the `version` suffix
- Keep snake case (?)
- Use single quotes for value